test/selenium-test/sign-in-widget/login.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, OnInit } from '@angular/core';
+import { Router, NavigationStart} from '@angular/router';
+
+import { OktaAuthService } from '@okta/okta-angular';
+import * as OktaSignIn from '@okta/okta-signin-widget';
+
+@Component({
+  selector: 'app-secure',
+  template: `
+    <!-- Container to inject the Sign-In Widget -->
+    <div id="okta-signin-container"></div>
+  `
+})
+export class LoginComponent {
+  authService;
+  widget = new OktaSignIn({
+    el: '#okta-signin-container',
+    baseUrl: 'https://dev-411042.okta.com',
```

Comment:
Can we pass the `baseUrl` and `clientId` as env vars so that it's easier to test with other orgs?

Reply:
@vijetmahabaleshwar-okta 
Now it passed as env vars. See my last changes.